vendor/github.com/vishvananda/netlink/nl/xfrm_state_linux.go

```diff
@@ -92,7 +92,7 @@ func (msg *XfrmStats) Serialize() []byte {
 
 type XfrmUsersaInfo struct {
 	Sel          XfrmSelector
-	Id           XfrmId
+	ID           XfrmId
```

Comment:
This is vendored code from a dependency, so should not be modified here (instead a pull request should be opened for that upstream in https://github.com/vishvananda/netlink/